opentreemap/treemap/migrations/0032_rename_to_role_default_permission_level.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('treemap', '0031_add_custom_id_to_default_search_fields'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='role',
+            old_name='default_permission',
+            new_name='default_permission_level',
+        ),
+    ]
```

Comment:
A simpler solution to avoid these headaches would be to just rename the model field, not the DB column:
```python
default_permission_level = models.IntegerField(
    db_column='default_permission',
    choices=FieldPermission.choices,
    default=FieldPermission.NONE)
```